common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java

```diff
@@ -93,7 +93,8 @@
  * This {@code classpath} configuration is only supported on YARN versions >= 2.9.0.
  */
 public class YarnShuffleService extends AuxiliaryService {
-  private static final Logger logger = LoggerFactory.getLogger(YarnShuffleService.class);
+  private static final Logger staticLogger = LoggerFactory.getLogger(YarnShuffleService.class);
+  private Logger logger = null;
```

Comment:
```suggestion
  private Logger logger = staticLogger;
```

And use logger for all non-static methods.
(See comment below in `serviceInit`).